pkgs/applications/emulators/wine-nine-standalone/nine.nix

```
@@ -0,0 +1,56 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, meson
+, ninja
+, libGL
+, libdrm
+, mesa
+, xorg
+, wine
+, wine64
+}:
+
+let
+  arch = if stdenv.is32bit then "32" else "64";
+in
+stdenv.mkDerivation rec {
+  pname = "wine-nine-standalone";
+  version = "0.10-unstable";
+
+  src = fetchFromGitHub {
+    owner = "iXit";
+    repo = pname;
+    #rev = "v${version}"
+    rev = "5ba57d60a351d87d3b2ef18acd8e43a15e2ece6b";
+    hash = "sha256-3b0irtj5aDEXkpOZNJgw2Ba5RXnYA90mcmncNILIszM=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    meson
+    ninja
+    (if stdenv.is32bit then wine else wine64)
+  ];
+  buildInputs = [ libGL libdrm mesa.driversdev xorg.libX11 ];
+
+  preConfigure = ''
+    ./bootstrap.sh --distro nixos
```

```suggestion
    ./bootstrap.sh --distro nixpkgs
```
technically, not sure if that is some magic value

---

The upstream bootstrap sources /etc/os-release to extract the ID string. This is where that value comes from. Since /etc/os-release isn't available in a build environment I sent a patch upstream to take a command line argument instead, so it could have been anything really.
https://github.com/iXit/wine-nine-standalone/pull/153